Cubical/Data/Int/Properties.agda

```diff
@@ -233,6 +205,73 @@ ind-assoc _·_ f g p q base m n (suc o) =
     where
     IH = ind-assoc _·_ f g p q base m n o
 
++-comm : ∀ m n → m + n ≡ n + m
++-comm m (pos n) = ind-comm _+_ pos sucInt refl sucInt+ +sucInt pos0+ m n
++-comm m (negsuc n) = ind-comm _+_ negsuc predInt refl predInt+ +predInt negsuc0+ m n
+
 +-assoc : ∀ m n o → m + (n + o) ≡ (m + n) + o
 +-assoc m n (pos o) = ind-assoc _+_ pos sucInt +sucInt refl (λ _ _ → refl) m n o
 +-assoc m n (negsuc o) = ind-assoc _+_ negsuc predInt +predInt refl +predInt m n o
+
+addEq : ℕ → Int ≡ Int
+addEq zero = refl
+addEq (suc n) = compPath (addEq n) sucPathInt
+
+subEq : ℕ → Int ≡ Int
+subEq n i = addEq n (~ i)
+
+_+'_ : Int → Int → Int
+m +' pos n    = transport (addEq n) m
+m +' negsuc n = transport (subEq (suc n)) m
+
+private
+  Lnegsuc : ∀ n z → z +' negsuc n ≡ z +negsuc n
+  Lnegsuc 0 z = refl
+  Lnegsuc (suc n) z = compPath (Lnegsuc n (predInt z)) (sym (predInt+negsuc n z))
+
+  Lpos : ∀ n z → z +' pos n ≡ z +pos n
+  Lpos zero z = refl
+  Lpos (suc n) z = cong sucInt (Lpos n z)
+  
+  Lemma : ∀ m n → m +' n ≡ m + n
+  Lemma m (pos n) = Lpos n m
+  Lemma m (negsuc n) = Lnegsuc n m
+
++'≡+ : _+'_ ≡ _+_
++'≡+ i m n = Lemma m n i
```

Comment:
Why not inline the proof of Lemma here? It seems a bit unnecessarily verbose to make a private module for a proof that is this short.

Reply:
To inline this proof elegantly I changed the definition of `subEq` a little bit. Otherwise I will have to do

```
+'≡+ i m (negsuc (suc n)) = compPath (λ i → (+'≡+ i (predInt m) (negsuc n))) (sym (predInt+negsuc n m)) i
```

The new definition make it as clean as the proof for the `pos` case:

```
+'≡+ i m (negsuc (suc n)) = predInt (+'≡+ i m (negsuc n))
```